src/themes/device.ts

```diff
@@ -1,3 +1,5 @@
+export type SizeType = keyof typeof size
+
 export const size = Object.freeze({
```

Comment:
The `size` object is frozen. Can we add a `Readonly` type on it since it's a ts file now, so it will prompt devs incase they try to override the `size`?